config/jenkins-jobs_infra.ci.jenkins.io.yaml

```diff
@@ -509,11 +509,22 @@ jobsDefinition:
             description: "docs.jenkins.io File Share Service Principal Writer"
             subscriptionId: "${JENKINSINFRA_AZURE_PRIMARY_SUBSCRIPTION_ID}"
             tenant: "${JENKINSINFRA_AZURE_TENANT_ID}"
+      stats.jenkins.io_previews:
+        name: stats.jenkins.io PR previews
+        allowUntrustedChanges: true
+        jenkinsfilePath: Jenkinsfile_previews
+        enableGitHubChecks: true
+  website-production-jobs:
+    name: Website Production Jobs
+    description: "Folder hosting all the Website jobs dedicated to deployment in production"
+    kind: folder
+    children:
       stats.jenkins.io:
         name: stats.jenkins.io
         allowUntrustedChanges: false
         jenkinsfilePath: Jenkinsfile
-        enableGitHubChecks: true
+        enableGitHubChecks: false
```

Comment:
Needs a follow-up pull request on stats.jenkins.io job to add a commit status like what's done for terraform in shared pipeline library function so contributors can know if the main branch build failed even without access to infra.ci.jenkins.io, without compromising security:
https://github.com/jenkins-infra/pipeline-library/blob/d59aa2ab110dd24ef0dd25df7f10a04a3f4f12e8/vars/terraform.groovy#L143-L148